src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const handleTabClick = (id, event) => {
+    event.preventDefault();
+    if (id !== activeTabId) {
+      onTabSelected(id);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTab.id ? 'is-active' : ''}
```

Comment:
To set classes conditionally it's better to use classnames library https://www.npmjs.com/package/classnames